renovate.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "baseBranches":[
+     "main",
+     "release-v0.20",
+     "release-v0.19",
+     "release-v0.18",
+     "release-v0.17"
+  ],
+  "constraints":{
```

Comment:
Not sure what the impact of this configuration

Reply:
I removed it